.github/dependabot.yml

```diff
@@ -4,7 +4,7 @@ version: 2
 updates:
   - package-ecosystem: docker
     # We use this file to load the last known good version of the image
-    directory: .config/
+    directory: .src/ansible_navigator/data
```

Comment:
Remove the dot here or it will not work. Also add the trailing slash like in original.

Reply:
done